### PR TITLE
Added repo link in footer

### LINKED ISSUE #30: Add repo link in footer
We should have the footer show the repo link like for other sites of ours:
image

### PR DESCRIPTION
**Pull request by akhilkh2000**
I added the footer to wallet.htm file as well

Closes #30

### CODE REVIEW

**Review by aman7870 — APPROVED** [review]

**Comment by aman7870** [comment]
Congratulations on your first PR in Real Dev Squad.

File: index.html [hunk 1]
```diff
@@ -108,6 +108,19 @@
       </div>
     </form>
 
+    <!-- footer for repo link -->
```
Comment (aman7870) [inline index.html:111]: Hi @akhilkh2000, please remove this comment as it's not that complex part people will get it by looking at the <footer> tag

**Comment by aman7870** [comment]
Can you please keep CSS in proper indentation.

File: index.html (hunk 1 above)
Comment (akhilkh2000) [reply index.html:111]: Okay. I will do this

**Comment by akhilkh2000** [comment]
> Can you please keep CSS in proper indentation.

My Vscode had messed up the indentation at first so in the latest commit I had re-used the original indentation that is why it shows changes from my previous identation mess up . Could you help me out with this if the indentation is still not okay ? How do I revert it back as my editor is showing it as original

**Pull request merged**